packages/front-end/components/Share/ShareModal.tsx

```diff
@@ -263,8 +263,7 @@ const ShareModal = ({
       }
       if (onSuccess && typeof onSuccess === "function") onSuccess();
       setLoading(false);
-      // @ts-expect-error TS(2722) If you come across this, please fix it!: Cannot invoke an object which is possibly 'undefin... Remove this comment to see the full error message
-      refreshList();
+      if (refreshList) refreshList();
```

Comment:
FYI - you could also do `refreshList?.()`

Reply:
Done